### PR TITLE
Memory2 javadoc fixes and Github matrix testing

### PR DESCRIPTION
**Pull request by davecromberge**
The Teletype `<tt>` tag has been deprecated in HTML 5 and was resulting in Javadoc generation errors on this branch.  The official HTML documentation recommends using alternatives such as `<samp />` or `<code />` instead.
This PR replaces `<tt />` usages with `<code />`.

Finally, matrix testing support has been added to the build to run the tests against Java 8-13.

### CODE REVIEW

File: .github/workflows/maven.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ name: Java Test Coverage with Maven, Coveralls
 on:
     pull_request:
     push:
-        branches: [ master ]
+        branches: [ master, Memory2-doc-fixes-gh-actions ]
```
Comment (leerho) [inline .github/workflows/maven.yml:6]: I assume this is temporary until we merge into master.

Comment (davecromberge) [reply .github/workflows/maven.yml:6]: Good spot, this was for testing and I'll remove it now before merging.

**Review by leerho — APPROVED** [review]

**Review by leerho — APPROVED** [review]

**Comment by leerho** [comment]
I will let you merge.

**Comment by davecromberge** [comment]
@leerho this merges into the Memory2 branch, a separate PR will be required for all the changes for Memory2.

**Pull request merged**